### ISSUE TITLE
Adding Lipsyncing for German would be fantastic!

### ISSUE THREAD

**Issue by JPhilipp** [met4citizen/TalkingHead#21]
There's currently English, Lithuanian and Finnish, but it would be fantastic to also have lipsync-de.mjs! I'm currently using this library for a host to a German ChatGPT-API-powered twitch chat.

Cheers!

**Comment by met4citizen** [comment]
Yes, that would be great! For more information on creating new lip-sync language modules, check out Appendix C in the README.

**Comment by 3bdop** [comment]
Hi @met4citizen, Your work is really amazing and I'm wondering if an Arabic lip-sync is also possible to do?

**Comment by 3bdop** [comment]
I was testing the alternative solution that you suggested, but there is an error with Arabic only with English its working with no issues, I don't know if I need to modify smth in the code so Arabic can work.

![Image](https://github.com/user-attachments/assets/f82104d8-9f70-4d16-81b4-7c2c7d105781) 
@met4citizen

**Comment by met4citizen** [comment]
It doesn't work because there is no Arabic voice configured in the test app hosted here on GitHub Pages.

If you want to test it using the test app (index.html), you'll need to clone the project to your own web server and add an Arabic voice entry to the `siteconfig.js` file. For example:

```javascript
...
microsoftVoices: {
  ...
  "qa-Amal": { lang: "ar-QA", id: "ar-QA-AmalNeural" },
 },
```

I tested this in my own environment, and it worked well (though I didn't understand a word that was said 🙂).
